glue/viewers/scatter/state.py

```diff
@@ -29,6 +29,8 @@ class ScatterViewerState(MatplotlibDataViewerState):
     dpi = DDCProperty(72, docstring='The resolution (in dots per inch) of density maps, if present')
     plot_mode = DDSCProperty(docstring="Whether to plot the data in cartesian, polar or another projection")
     angle_unit = DDSCProperty(docstring="Whether to use radians or degrees for any angular coordinates")
+    flip_lonaxis = DDCProperty(False, docstring="Whether or not to flip x coordinates; used for full-sphere projections")
+    flip_lataxis = DDCProperty(False, docstring="Whether or not to flip y coordinates; used for full-sphere projections")
```

Comment:
Just to make sure i understand, how is using these options different to setting an ``x_min`` value larger than ``x_max``? If it's different, is there any scope to give these more general names? (e.g. ``flip_x`` and ``flip_y``)

Reply:
Using these has the same effect as `x_min` > `x_max`, in that the axis will run in the opposite direction to usual. I added these in because matplotlib doesn't allow us to change bounds for the full-sphere projections. For the other projections one can just use the bounds instead. But maybe `flip` names are more future-proof, in case we someday have other bound-less projections?

Reply:
I wonder if we could do this without adding new callback properties - what if we just use x_min and x_max but at the point of setting the limits, if one can't actually set the full limits we would still use the order of x_min/x_max to determine whether to flip the limits in the plot. The benefit of this is that if a user uses the flip buttons in the existing axes limits in the UI it should still work?